queries.sql

```diff
@@ -13,3 +13,29 @@ SELECT id, name, escape_attempts, weight_kg FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name NOT IN ('Gabumon');
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
```

Comment:
- [ Required ] I think it would be nice if you make a `SELECT` query to verify if the `species` column was updated before performing the `ROLLBACK`. In the project requirements, you are required to verify that the change was effected before performing the `ROLLBACK`. Please see the screenshot below and rectify this.

## Project requirements
![Screenshot from 2023-09-29 15-17-56](https://github.com/kazim110/Vet-Clinic/assets/16392046/86f46f98-d398-4a20-bc21-384015f084b5)